frontend/requests/analyse_hierarchy_request_dialog.html

```diff
@@ -1,88 +1,82 @@
-<md-dialog flex-sm="80" flex-xs="100" flex-gt-sm="50">
+<md-dialog class="analyse_request__dialog" flex-sm="80" flex-xs="90" flex-gt-sm="50">
```

Comment:
remember that on desktop screen size, it should remain the same thing.

Reply:
added max-height